src/KubernetesClient/Kubernetes.ConfigInit.cs

```diff
@@ -41,9 +42,8 @@ public Kubernetes(KubernetesClientConfiguration config, HttpClient httpClient) :
         public Kubernetes(KubernetesClientConfiguration config, HttpClient httpClient, bool disposeHttpClient) : this(httpClient, disposeHttpClient)
         {
             ValidateConfig(config);
-            CaCerts = config.SslCaCerts;
-            SkipTlsVerify = config.SkipTlsVerify;
```

Comment:
It looks like this is getting dropped?

Reply:
The config object is saved below in `this.config = config`, and they are accessible from there.

Reply:
any detail reason for this change?

Reply:
I changed it to save the entire config object rather than just two fields out of it, because it was useful to have it when instantiating a request - in particular to construct the credentials. Now that I use the ServiceCredentials object from the Kubernetes client, this can probably be reverted. But it's not like any information is being lost here. :-)